Docs number text date component (#1892)
Altinn/altinn-studio-docs: content/altinn-studio/reference/ux/components/Number/_index.en.md

## Configuration

### Add component

```json{hl_lines="6-"}
 {
        "id": "number1",
        "type": "Number",
        "textResourceBindings": {
          "title": "Amount"
        },
        "value": "100000",
        "direction": "vertical",
        "icon": "https://www.example.com/icon.png",
        "formatting": {
          "currency": "NOK",
          "position": "suffix",
          "number": {
            "thousandSeparator": " "
          }
        },
      },
```

### Add title and elements

<br>

#### `textResourceBindings.title`

The title for the text is shown as a label and can be added as text directly or referenced via a text key to a [text resource](/nb/altinn-studio/reference/ux/texts/#legge-til-og-endre-tekster-i-en-app).

#### `value`

Displays the value for the label and can be added as a string or an expression.

#### `direction`

> Decides if the text should is shown under or next to the label. horizontal for "next to" and vertical for "under". 

Reviewer: ```suggestion
Decides if the number should be shown under or next to the label. `horizontal` for showing it next to the label and `vertical` for under. 
```
